src/package/nuspec/TestPlatform.TestHost.nuspec

```diff
@@ -14,11 +14,18 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <dependencies>
       <group targetFramework="netcoreapp1.0">
-      <dependency id="Microsoft.TestPlatform.ObjectModel" version="$Version$"/>
+        <dependency id="Microsoft.TestPlatform.ObjectModel" version="$Version$"/>
         
-      <!-- CommunicationUtilities requires 8.0.3, but 8.0.3 doesn't provide a netcoreapp1.0 target -->
-      <dependency id="Newtonsoft.Json" version="9.0.1"/>
-      <dependency id="Microsoft.Extensions.DependencyModel" version="1.0.3"/>
+        <!-- CommunicationUtilities requires 8.0.3, but 8.0.3 doesn't provide a netcoreapp1.0 target -->
+        <dependency id="Newtonsoft.Json" version="9.0.1"/>
+        <dependency id="Microsoft.Extensions.DependencyModel" version="1.0.3"/>
+      </group>
+
+      <group targetFramework="uap10.0">
+        <dependency id="Newtonsoft.Json" version="9.0.1"/>
```

Comment:
Check what we can remove from here? Add dependency on ObjectModel.

Reply:
added dependency on object model, removed what we could